README.md

```diff
@@ -63,18 +68,16 @@ const userSchema = z.object({
 @Router()
 class UserRouter {
   constructor(
-    @Inject(UserService) private readonly userService: UserService
-  ) {
-  }
+    private readonly userService: UserService
+  ) {}
 
   @Procedure(ProtectedProcedure)
   @Query({ output: z.array(userSchema) })
   async getUsers() {
     try {
-      const users = await this.userService.getUsers();
-      return users;
-    } catch (e: unknown) {
-      throw new TRPCError("An error has occured when trying to get users.", "INTERNAL_SERVER_ERROR", e)
+      return this.userService.getUsers();
+    } catch (error: unknown) {
+      throw new TRPCError("An error has occured when trying to get users.", "INTERNAL_SERVER_ERROR", error)
```

Comment:
In general, the way I handle errors here isn't correct, see https://trpc.io/docs/v10/server/error-handling
I need to pass an object rather then single parameters here.